### ISSUE TITLE
A11y_SQL Azure Data Studio _Export query results_Keyboard: Tooltips of save as csv, save as excel etc. buttons is not accessible with keyboard.

### ISSUE THREAD

**Issue by sohelmulani1998** [microsoft/azuredatastudio#15842]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Windows10
Screenreader: JAWS, NVDA

**Repro Steps:**

1. Launch Azure Data Studio.
2. Tab Navigate to File and press enter then tab till New Query and hit enter.
3. Connect to a Server in the Query Editor and choose the credential, and optionally the database to connect to
4. Type a SQL query statement that produces a result set
5. Tab till Run and press enter
6. Tab Navigate to the Results tab. 
8. From the side menu, Tab navigate to "Save As <file type>" buttons , where <file type> is either CSV, Excel, JSON, XML. 
9. Check if tooltip of those buttons are accessible with keyboard.  

**Actual:**
Tooltips of save as csv, save as excel etc. buttons is not accessible with keyboard.

**Expected:**
Tooltips of save as csv, save as excel etc. buttons should be accessible with keyboard.

**User Impact:**
Keyboard only user will face difficulty while accessing the controls as they wont be able to access the tooltip with keyboard.

**Comment by alanrenmsft** [comment]
this is a general problem: https://github.com/microsoft/vscode/issues/52016

**Comment by Ashutoshsahoo164** [comment]
As per the above mentioned bug , the bug is closed. Hence closing this bug.

GitHubTags: #Closed;
